vue-app/src/views/Verified.vue

```diff
@@ -11,7 +11,7 @@
           <span class="emoji">🎉</span>
           <div class="flex-title">
             <h1>Ready to contribute!</h1>
-            <links v-if="txHash" :to="blockExplorerUrl">
+            <links v-if="$route.params.hash" :to="blockExplorerUrl">
```

Comment:
Would it make sense to switch this to TransactionReceipt for consistency around other transactions (reallocation, registration, contribution)

Reply:
Agreed, let's use that component here.